Mono.Cecil.Cil/Symbols.cs

```diff
@@ -761,6 +761,44 @@ public interface ISymbolReaderProvider {
 		ISymbolReader GetSymbolReader (ModuleDefinition module, Stream symbolStream);
 	}
 
+#if !NET_CORE
+	[Serializable]
+#endif
+	public sealed class SymbolNotFoundException : FileNotFoundException {
```

Comment:
minor: there's an inconsistency between `Symbol` and `Symbols` in the API
e.g. `GetSymbolReader` and `ReadSymbols`
I think it's more the later (plural) but I don't care very much - still you should pick an official *winner* before adding more :)